docs/editors/settings.md

```diff
@@ -497,9 +497,9 @@ Rules to disable by default. See [the documentation](https://docs.astral.sh/ruff
     }
     ```
 
-### `extendIgnore`
+### `unfixable`
```

Comment:
I think you meant to add a new `unfixable` section, right? It seems like this is replacing the `extendIgnore` section.

Reply:
It was intentional to remove though unrelated to the PR. Is extendIgnore supported? I couldn’t find any reference to it in the server crate.

Reply:
Oh, that's correct. Not sure why I thought there's a `extendIgnore` when I wrote the docs. Thanks